### DISCUSSION TITLE
appMetrica ANE

### DISCUSSION THREAD

**Discussion by MESepehr** [airsdk/Adobe-Runtime-Support#1775] [General]
Hi Michael.
Do you have a plan to develop appmetrica ANE?

https://appmetrica.yandex.com/about

@marchbold

**Comment by marchbold** [comment]
Hi, Currently we don't have any plans to develop an extension for this service. You can contact our custom development team (support@airnativeextensions.com) to get one developed or you can add an idea to our uservoice site: https://distriqt.uservoice.com/
